index.html

```diff
@@ -47,119 +54,131 @@ <h4>LET'S CONNECT</h4>
                     </li>
                 </ul>
             </div> 
-        </section>
-        <section class="works-container">
+            </section>   
+        </div>
+
+        <section class="works-container" id="works-container">
             <div class="flex-card">
                 <img class="snapshot" src="SnapshootPortfolio.svg" alt="portfolio 1">
-                <h1>Tonic</h1>
-                <ul>
-                    <li class="element1">CANOPY</li>
-                    <li><img src="Counter.svg" alt="dot"></li>
-                    <li class="element2">Back End Dev</li>
-                    <li><img src="Counter.svg" alt="dot"></li>
-                    <li class="element2">2015</li>
-                </ul>
-                <p>A daily selection of privately personalized reads; 
-                    no accounts or sign-ups required.</p>
-                <ul class="bottom-list">
-                    <li><button class="language-buttons" type="button">html</button></li>
-                    <li><button class="language-buttons" type="button">css</button></li>
-                    <li><button class="language-buttons" type="button">javaScript</button></li>
-                </ul>
-                <button class="p-button" type="button">See project</button>
+                <div class="column">
+                    <h1><strong>Tonic</strong></h1>
```

Comment:
- [ ] As the previous reviewer suggested, kindly increase the font weight of all your project card heading to match the Figma design 🤗.

Your design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/188228923-ed68d993-849f-4685-b293-6847e9bbeecc.png)

Figma design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/188229056-cc7796ea-75c0-452a-902d-47205c451316.png)